docs/en/index.rst

```diff
@@ -46,7 +47,10 @@ Example of configuring the authentication middleware using ``authentication`` ap
         {
             $service = new AuthenticationService();
             $service->setConfig([
-                'unauthenticatedRedirect' => '/users/login',
+                'unauthenticatedRedirect' => Router::url([
```

Comment:
IMO we shouldnt need the Router::url() call here in config, but lazy applied later on in code.

Reply:
If the code doesnt handle arrays here yet, we should fix also the code.

Reply:
This method would be called when the AuthenticationMiddleware is called which should be after the Routing middleware is called which means Router::url() will work.

Reply:
Jusr array not working,

Reply:
We should fix it so that array Syntax suffices and works.